simple-codecs/src/test/scala/akka/stream/alpakka/recordio/RecordIOFramingSpec.scala

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.recordio
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.alpakka.recordio.scaladsl.RecordIOFraming
+import akka.stream.scaladsl.{Flow, Keep, Sink, Source}
+import akka.stream.scaladsl.Framing.FramingException
+import akka.testkit.TestKit
+import akka.util.ByteString
+import org.scalatest.{FlatSpecLike, Matchers}
+import org.scalatest.concurrent.ScalaFutures
+
+import scala.collection.immutable.Seq
+
+class RecordIOFramingSpec(_system: ActorSystem)
+    extends TestKit(_system)
+    with FlatSpecLike
+    with Matchers
+    with ScalaFutures {
+  def this() = this(ActorSystem("RecordIOFramingSpec"))
```

Comment:
the ActorSystem must also be terminated (`TestKit.shutdown`) at the end (AfterAll)